lib/active_support/cache/redis_cluster_store.rb

```diff
@@ -16,7 +16,7 @@ def increment(key, amount = 1, options = {})
         ttl = _expires_in(options)
         normalized_key = normalize_key(key, options)
         instrument(:increment, key, :amount => amount) do
-          with do |c|
+          redis.with do |c|
```

Comment:
This is a public method on this redis cache class. 👍 
https://api.rubyonrails.org/classes/ActiveSupport/Cache/RedisCacheStore.html#method-i-redis